lib/mix/tasks/ash_authentication.add_strategy.ex

```diff
@@ -426,6 +427,7 @@ if Code.ensure_loaded?(Igniter) do
           end
 
           defp body(params) do
+            # TODO: Set to your magic link acceptance URL.
             url = url(~p"/auth/user/magic_link/?token=\#{params[:token]}")
```

Comment:
Will this line up with the default? We shouldn't have a TODO that the user has to do post-install hidden in a file. We also changed the URL for `confirmation`, I think we should change it in the same way for this, so that it is something like `/magic_link`. So we should update this to match that suggestion. In the warning, we should tell users to update the URL to be what the old default is in order to upgrade, i.e `path: "/auth/user/magic_link", token_as_route_param?: false`, or link to a guide that explains what to do, because just changing `require_interaction?` to `true` is not enough (they also have to modify their router, and make these changes etc.

Reply:
it will work with the default plug-based URL. it should be changed if they're using the new route in AAP though.

Reply:
Right, so I think we should default to the one that works w/ Phoenix since that is 99% of installations.

Reply:
turns out we have this awesome variable in scope `use_web_module` :)